src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -1962,6 +2054,49 @@ describe('SavedObjectsRepository', () => {
 
       expect(onBeforeWrite).toHaveBeenCalledTimes(1);
     });
+
+    it('accepts a custom refresh setting', async () => {
+      await savedObjectsRepository.update(
+        'globaltype',
+        'foo',
+        {
+          name: 'bar',
+        },
+        {
+          refresh: true,
+          namespace: 'foo-namespace',
+          references: [
+            {
+              name: 'ref_0',
+              type: 'test',
+              id: '1',
+            },
+          ],
```

Comment:
nit: since these are optional properties, we can rather leave them out of the test to make it a bit more compact